add embedding bag skeleton take 2

hyuen opened the pull request:
Summary: adding skeleton implementations of quantized embedding tables with zeroes

Test Plan: compilation, farm test, and ran test_find_dangling_impls and passed

Differential Revision: D29518274

walterddr commented on test/test_dispatch.py:
```diff
@@ -888,6 +888,9 @@ def test_defaultbackend_math(self):
+        dangling_impls = C._dispatch_find_dangling_impls()
+        if len(dangling_impls) > 0:
+            print(dangling_impls)
         self.assertEqual(0, len(C._dispatch_find_dangling_impls()))
```
nit: you can actually modify the asserEqual(msg=) to stringify the output for the assert failure
```suggestion
        self.assertEqual(0, len(C._dispatch_find_dangling_impls()), msg=f'Expect zero dangling impls, but found: {dangling_impls}')
```

Commit:
add embedding bag skeleton take 2 (#61126)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/61126

adding skeleton implementations of quantized embedding tables with zeroes

Test Plan: compilation, farm test, and ran test_find_dangling_impls and passed

Reviewed By: walterddr

Differential Revision: D29518274

fbshipit-source-id: 48f41c604c56c1e3b8f69b75a9b98d0afa5bfff2
